client/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+	"name": "Electrode Boilerplate",
+	"icons": [
+		{
+			"src": "./icons/android-chrome-192x192.png",
```

Comment:
Thanks to the webpack plugin we're using to load the manifest file, we can just specify the paths like we would if we're requiring them, and it will bundle them + replace with the new path for us.